aip/client-libraries/4234.md

```diff
@@ -0,0 +1,103 @@
+---
+aip:
+  id: 4234
```

Comment:
I might ask that we re-number this, but unsure to what yet. Putting it in client libraries makes sense, though.

Reply:
Sure. Just LMK.

Reply:
Let's do 4291.